botorch/models/utils.py

```diff
@@ -133,20 +133,31 @@ def check_no_nans(Z: Tensor) -> None:
 
 
 def check_min_max_scaling(
-    X: Tensor, strict: bool = False, atol: float = 1e-2, raise_on_fail: bool = False
+    X: Tensor,
+    ignore_dims: Optional[List[int]] = None,
```

Comment:
Mind adding this as the last arg so that in case folks call this with positional args it doesn't break stuff?

Reply:
Yep, that makes sense.